actionmailer/lib/action_mailer/base.rb

```diff
@@ -342,6 +342,27 @@ module ActionMailer
   # using <tt>before_action</tt> rather than <tt>after_action</tt> in your
   # Action Mailer classes so that headers are parsed properly.
   #
+  # = Rescue
+  #
+  # Handle exceptions raised from within the <tt>mail</tt> method with <tt>rescue_from</tt>.
+  #
+  # Suppose a third party provider is used to send emails.
+  # It can raise a ThirdPartyProvider::InvalidEmail exception.
+  #
+  # Here's how you can use <tt>rescue_from</tt> to intercept all ThirdPartyProvider::InvalidEmail.
```

Comment:
What about leading with the reasons for not using a `rescue` block?

```suggestion
  # +rescue+ blocks inside of a mailer method cannot rescue errors that occur
  # outside of rendering -- for example, record deserialization errors in a
  # background job, or errors from a third-party mail delivery service.
  #
  # To rescue errors that occur during any part of the mailing process, use
  # {rescue_from}[rdoc-ref:ActiveSupport::Rescuable::ClassMethods#rescue_from]:
```